config/relay.go

```diff
@@ -14,6 +14,9 @@ type Relay struct {
 
 	// Backoff configuration.
 	Backoff Backoff `json:"backoff"`
+
+	//Timeout interval, Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+	Timeout	string	`json:"timeout"` 
```

Comment:
mind changing this to an `int` in seconds for now? Later I'll add a `Duration` type which allows numeric seconds _and_ the Go style string units, but for now int is more uniform.

On a side note make sure to `gofmt` :D `// Timeout`